### ISSUE TITLE
Upgrade from 4.3.10 to 4.3.20 fails

### ISSUE THREAD

**Issue by frapell** [plone/plone.app.upgrade#246]
I was trying to upgrade a 4.3.10 from a customer to 4.3.19 (And later tried 4.3.20 and I was getting the same), where it would fail with this:

```
2020-09-22 19:16:44 ERROR plone.app.upgrade Upgrade aborted. Error:
Traceback (most recent call last):
  File "/opt/plone/eggs/Products.CMFPlone-4.3.20-py2.7.egg/Products/CMFPlone/MigrationTool.py", line 285, in upgrade
    step['step'].doStep(setup)
  File "/opt/plone/eggs/Products.GenericSetup-1.8.11-py2.7.egg/Products/GenericSetup/upgrade.py", line 166, in doStep
    self.handler(tool)
  File "/vagrant/src/plone.app.upgrade/plone/app/upgrade/v43/final.py", line 358, in fix_double_smaxage
    if registry.records[maxage].field.recordName != def_smaxage:
AttributeError: 'Int' object has no attribute 'recordName'
```

Now, I haven't had time to check if this is happening with a vanilla Plone 4.3.10

**Comment by frapell** [comment]
@mauritsvanrees Should I submit a PR for this? It does work for my case...

**Comment by mauritsvanrees** [comment]
We won't do any Plone 4.3 releases anymore, so I would say: don't bother.

But it's not forbidden. And I am okay with making an occasional package release, even if it won't end up in an official Plone release.

**Comment by frapell** [comment]
@mauritsvanrees Yeah... I was mostly thinking in someone else facing this issue, they could pin a newer plone.app.upgrade and continue with the upgrade.

I will create a PR

**Comment by ale-rt** [comment]
This looks like it is fixed by #247.
Please reopen if not

**Comment by mauritsvanrees** [comment]
I have just made a release with that fix: `plone.app.upgrade` 1.4.8. It won't make its way to an official Plone release, but if people need it it is there now.
